### DISCUSSION TITLE
Firefly Desktop 2.0.0 Beta 3

### DISCUSSION THREAD

**Discussion by nicole-obrien** [iotaledger/firefly#4179] [Announcements]
### Changelog
- Downgrade electron builder to fix install issues on windows
- Fix issue where node settings would wipe when visiting advanced settings page causing user not to be able to login
- Fix amount input for , vs . decimal point
- Fix unit selection on send form to allow sending glow
- Improve the ui of the asset dropdown to make it easier to find assets on send form
- Add gift storage deposit to native token send flow
- Implement virtual lists to improve rendering speed of wallet tab
- Add filters to asset list
- Improve activity generation to avoid issues with claim status resetting
- Sort activities groups correctly by month
- Add filters to activity list

### Known Bugs:
- Sending 0 value transactions with metadata and tag fails
- Reseting asset metadata may crash the profile until restart
- Hiding asset does not display data correctly on the popup
- Minting activity still not showing correctly in the activity list
- Storage deposit calculation for minting transaction is wrong
- Showing hidden activities does not show rejected transactions

### File Hashes
[How to verify the authenticity of your Firefly Desktop download](https://wiki.iota.org/chrysalis-docs/firefly/verify_download)
| File | Platform | SHA256 Hash |
| --- | --- | --- |
| firefly-desktop-2.0.0-beta-3.exe | Windows | 30b56657fcdb1d56b30fd96dc7292ee0aab234a854e1b159d34f0d7a17e18dbe |
| firefly-desktop-2.0.0-beta-3.AppImage | Linux | b4a1cc17290b81932a5ac60c50f2619d787bf4c9bc80f1261e34a34694dc239a |
| firefly-desktop-2.0.0-beta-3.dmg | MacOS |  ac5048679d61f5463c94b801268a408174e4ec6ebf04963c2d2dc984c9a08b5c |

<hr /><em>This discussion was created from the release <a href='https://github.com/iotaledger/firefly/releases/tag/desktop-2.0.0-beta-3'>Firefly Desktop 2.0.0 Beta 3</a>.</em>

**Comment by a673130565** [comment]
Windows 7 is not available, please debug well. Every version is the same problem。
